Formula/polynote.rb

```diff
@@ -0,0 +1,52 @@
+class Polynote < Formula
+  desc "The polyglot notebook with first-class Scala support"
+  homepage "https://polynote.org/"
+  url "https://github.com/polynote/polynote/releases/download/0.2.17/polynote-dist-2.12.tar.gz"
```

Comment:
Which version is it? 0.2.17 or 2.12?

Reply:
The `2.12` part of the file name references the Scala version it's compiled for. I changed the URL to use a distribution that doesn't have this version in the file name, to avoid confusion.